trafilatura/xpaths.py

```diff
@@ -144,7 +144,7 @@
     or contains(@class, "message-container") or contains(@id, "message_container")
     or contains(@class, "yin") or contains(@class, "zlylin") or
     contains(@class, "xg1") or contains(@id, "bmdh")
-    or @data-lp-replacement-content]''',
+    or @data-lp-replacement-content][not(starts-with(@class,'hljs'))]''',
```

Comment:
I added it to support this line: https://github.com/adbar/trafilatura/pull/401/files/df66b25c2319af203acb43cac648803e65de3351#diff-abef8bfc97e60c84035950c7da832f5ca07580db79e4fb9e77ef0f8b82112b30R499.
I want to be able to find span elements that have a class that starts `hljs` because it means it's a code block element.
I agree that it's not ideal and see you also commented on the other comment, so maybe if we can find better place for the implementation it can make this change unnecessary

Reply:
I need to think about it a bit more...